libyul/YulStack.cpp

```diff
@@ -273,7 +274,8 @@ YulStack::assembleWithDeployed(std::optional<std::string_view> _deployName)
 				{{m_charStream->name(), 0}}
 			);
 		}
-		creationObject.ethdebug["not yet implemented @ MachineAssemblyObject::ethdebug"] = true;
+		if (debugInfoSelection().ethdebug)
+			creationObject.ethdebug = evmasm::ethdebug::program(creationObject.assembly->name(), 0, creationObject.assembly.get(), creationObject.bytecode.get());
```

Comment:
What about `deployedObject`? You do access its `ethdebug` field in `StandardCompiler` but you're not filling it out here.

Reply:
Also, just FYI, you should be aware of this yet unfixed bug: #15745.